src/components/sign-in-or-create/index.njk

```diff
@@ -17,7 +17,7 @@
 {% block content %}
     {% include "common/errors/errorSummary.njk" %}
 
-    <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-3">{{ 'pages.signInOrCreate.mandatory.header' | translate }}</h1>
+    <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-3">{{ 'pages.signInOrCreate.mandatory.header' | translate | striptags | safe }} </h1>
```

Comment:
this is applied to the header rather than the title, was that the intent?

Reply:
Yes, the header is the intent, the title should remain unchanged. The non breaking space should be in the header so the 2 words go together. There is no need for a &nbsp in the title